pages/docs/support/upgrading-grouparoo/v02-v03.mdx

```diff
@@ -0,0 +1,86 @@
+   - If using the **Enterprise UI**, no additional steps are required. We have provided migrations for affected plugins that will be automatically applied the next time Grouparoo starts up.
+   - If using **Code Config**, you may need to specify which `syncMode` your destinations will be operating in. For more information, keep reading the following sections.
+
+### Setting sync modes in code config
+
+Config files for destinations now have an additional `syncMode` property to specify which mode the destination should operate in. From this point onwards, generating new destinations that support sync modes should automatically include the `syncMode` property in their config file. However, setting a sync mode for pre-existing destinations requires adding the property to the config file manually.
+
+Below is an example of the `syncMode` property being added to a Salesforce destination:
+
+```js
+{
+      id: "...",
+      name: "...",
+      class: "destination",
+      type: "salesforce-objects-export",
+      appId: "...",
+      groupId: "...",
+
+      syncMode: "sync", // How should Grouparoo sync with this destination? Options: "sync", "additive", "enrich"
+
+      options: {
+        profileObject: "Contact",
+        profileMatchField: "Email",
+        ...
+```
+
+Most destinations have default sync modes that will keep your destinations working just as before without any intervention from you. However, if the destination supports sync modes we generally recommend setting it explicitly in its configuration file. If a destination has multiple options for sync modes and it's falling back to a default, you may see warnings like these in the logs to remind you to set a sync mode:
+
+```
+warning: Using default sync mode "sync" for destination "pipedrive". You should explicitly set a sync mode for the destination to prevent unintended behavior.
+```
+
+### Choosing a sync mode
+
+It's important to understand what these sync modes mean. Our [blog post on Sync Modes](/blog/sync-modes) introduces the concept and explains what each available mode does.
+
+Not all destinations support all sync modes. To check which options are available and what the default mode is for a destination, you can use our cli with `--describe`:
+
+```
+$ grouparoo generate pipedrive:destination --describe
+
+🦘 Grouparoo: generate pipedrive:destination
+
+Config for a pipedrive Destination
+
+Required Arguments:
+  * id (required) - The id of this new Destination
+
+Required Options:
+  * parent (required) - The id of the pipedrive App to use for this Destination, e.g: `--parent data_warehouse`
+
+Optional Options:
+  * syncMode - How should profiles sync to the destination? e.g. `--sync-mode additive`. Options: sync, additive, enrich (default: "sync")
+```
+
+The last line of the previous example shows available options and default sync mode for the Pipedrive destination.
+
+### Migrating plugins that previously supported sync modes
+
```

Comment:
... this is still just for those using code config, yes?

Reply:
yes! Changed the title to an h4 but let me know if it's still unclear

Reply:
I think this covers it!

> **Note that these steps are automatically performed for you if you have been using the UI to configure your destinations.**